lib/dal/src/workspace_snapshot/graph.rs

```diff
@@ -1030,7 +1029,11 @@ impl WorkspaceSnapshotGraph {
             },
         );
         let filename_no_extension = format!("{}-{}", Ulid::new().to_string(), suffix);
-        let mut file = File::create(format!("/home/zacharyhamm/{filename_no_extension}.txt"))
+
+        let home_str = std::env::var("HOME").unwrap();
```

Comment:
We should probably not panic here.

Reply:
Given that this bit is debugging code, the `unwrap()` is probably fine.